libs/features/georoutingv2/georoutingv2.css

```diff
@@ -151,3 +151,9 @@
     max-width: 720px;
   }
 }
+
+@media only screen and (max-device-width: 956px) and (orientation: landscape) {
```

Comment:
Is there a reason to have this limitation to begin with? Could we not have this style for every device below desktop? Accessibility-wise, we should get some input as well. I tried mimicking a situation where the georouting modal overflows and:

The CTAs are not visible in the default view:
![main--cc--adobecom hlx page_drafts_devashish_vh (1)](https://github.com/user-attachments/assets/4a13b845-0d8e-4d4d-9d66-624bf8b2e9ba)

The close button is not visible when scrolled down:
![main--cc--adobecom hlx page_drafts_devashish_vh](https://github.com/user-attachments/assets/2b8b9a0e-3f34-4854-a293-e1f8e7d35d85)

Reply:
There is an exception issue specifically with iPhone devices, which is why this fix hasn’t been applied universally.

For more details, please refer to this discussion: 
https://github.com/orgs/adobecom/discussions/2895

This is the only solution I found to be viable for this issue.

Reply:
I had a quick read through the discussion and I agree that the additional dropdown should expand upwards.

But I still don't see the reason for not applying the `overflow: auto;` property universally. I understand that the issue is just on iPhone devices, but why create an exception with a media query value that most people will not understand refers to an iPhone if we could simply apply the fix universally? There might be arguments against that, but I'd like to ensure that option has been explored before going down this path.